test/Makefile

```diff
@@ -106,7 +126,10 @@ $(INITRAMFS)/usr/local:
 $(INITRAMFS)/test:
 	@make --no-print-directory -C apps
 
-$(INITRAMFS)/benchmark: | $(INITRAMFS)/benchmark/bin
+$(INITRAMFS)/benchmark: | $(INITRAMFS)/benchmark/bin $(INITRAMFS)/benchmark/benchmarks
+
+$(INITRAMFS)/benchmark/benchmarks:
+	@mkdir -p $@
 	@cp -rf $(CUR_DIR)/benchmark/* $@
```

Comment:
Is there a reason to modify the original benchmark directory structure? Originally, the lmbench binaries were placed under `benchmark/lmbench`, and the iperf3 binaries under `benchmark/iperf3`, which I believe is well-organized. However, after this PR, all benchmark binaries will be located in `benchmark/bin`, and a new `benchmark/benchmarks` directory has been introduced. Does this change offer any advantages?